visiont/scripts/train.py

```diff
@@ -89,27 +100,50 @@ def main(args):
         torch.backends.cudnn.benchmark = True
 
     # ImageNet stats for now
-    mean, std = [0.485, 0.456, 0.406], [0.229, 0.224, 0.225]
-
-    transform = Compose([
-        Convert("RGB"),
-        Resize(1024),
-        RandomCrop(224),
-        RandomHorizontalFlip(p=0.5),
-        RandomApply([ColorJitter(brightness=0.8, contrast=0.8, saturation=0.8, hue=0.2)], p=0.8),
-        RandomApply([GaussianBlur((3, 3), (1.5, 1.5))], p=0.1),
-        RandomGrayscale(p=0.2),
-        ToTensor(),
-        Normalize(mean=mean, std=std),
-        ToPatches(16)])
+    # mean, std = [0.485, 0.456, 0.406], [0.229, 0.224, 0.225]
```

Comment:
Why no `Normalize` anymore? We now feed in un-normalized images? Do you plan on getting the Open Images stats then?

Reply:
Token representations in transformers are not whitened `(mean=0, std=1)` not sure why we need them here.

Reply:
I don't follow; the token gets only cat-ed to the patch projections here, no?

https://github.com/daniel-j-h/visiont-pytorch/blob/4d9c4213aea9be291bd4f1b7708e2f103a639c52/visiont/models.py#L51